code/modules/atmos_automation/implementation/vent_pump.dm

```diff
@@ -88,4 +88,202 @@
 					injector_names|=I.id_tag
 			vent_pump = input("Select a vent:", "Vent Pumps", vent_pump) as null|anything in injector_names
 			parent.updateUsrDialog()
-			return 1
\ No newline at end of file
+			return 1
+
+/datum/automation/set_vent_pump_pressure//controls the internal/external pressure bounds of a vent pump.
+	name = "Vent pump: Pressure settings"
+
+	var/vent_pump = null
+	var/intpressureout = 0//these 2 are for DP vents, if it's a unary vent you're sending to it will take intpressureout as var
+	var/intpressurein = 0
+	var/extpressure = 0
+	var/mode = 0//0 for unary vents, 1 for DP vents.
+
+	Export()
```

Comment:
Ideally they should all be absolute pathed.
One issue that I can see arising from this is a bunch of similarly written code being incredibly hard to read and easy to break.